### ISSUE TITLE
Uncaught TypeError: Cannot read property 'dispose' of null

### ISSUE THREAD

**Issue by cometeer** [jeremyramin/terminal-plus#88]
Open 2 terminal windows
Switch between them by clicking the button (the one you can give a color)
Try to close one of the terminals

Additional info:
Prior to this the terminals stacked instead of switching. A reload fixed the issue but it came back once more. Maybe just a fluke?

**Atom Version**: 1.2.0
**System**: Microsoft Windows 10 Home
**Thrown From**: [terminal-plus](https://github.com/jeremyramin/terminal-plus) package, v0.14.1

### Stack Trace

Uncaught TypeError: Cannot read property 'dispose' of null

```
At /C:/Users/Tristan/.atom/packages/terminal-plus/lib/status-icon.coffee:95

TypeError: Cannot read property 'dispose' of null
    at status-icon.module.exports.StatusIcon.destroy (file:///C:/Users/Tristan/.atom/packages/terminal-plus/lib/status-icon.coffee:63:28)
    at TerminalPlusView.module.exports.TerminalPlusView.destroy (file:///C:/Users/Tristan/.atom/packages/terminal-plus/lib/view.coffee:145:17)
    at HTMLButtonElement.<anonymous> (C:\Users\Tristan\AppData\Local\atom\app-1.2.0\resources\app.asar\node_modules\atom-space-pen-views\node_modules\space-pen\lib\space-pen.js:220:36)
    at HTMLButtonElement.jQuery.event.dispatch (C:\Users\Tristan\AppData\Local\atom\app-1.2.0\resources\app.asar\node_modules\jquery\dist\jquery.js:4435:9)
    at HTMLButtonElement.elemData.handle (C:\Users\Tristan\AppData\Local\atom\app-1.2.0\resources\app.asar\node_modules\jquery\dist\jquery.js:4121:28)
```

### Commands

```
     -1:26.6.0 command-palette:toggle (atom-text-editor.editor.is-focused)
     -1:24.6.0 core:move-down (atom-text-editor.editor.mini.is-focused)
     -1:23.5.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -1:23.5.0 browser-plus:open (atom-text-editor.editor)
```

### Config

```json
{
  "core": {
    "disabledPackages": [
      "markdown-format",
      "linter-jshint"
    ],
    "autoHideMenuBar": true
  }
}
```

### Installed Packages

```coffee
# User
.bin, vundefined
atom-handlebars, v1.2.0
atom-ternjs, v0.8.4
browser-plus, v0.0.54
data-atom, v0.14.1
language-tsql, v0.19.0
linter, v1.11.1
markdown-scroll-sync, v0.4.2
meteor-api, v2.20.0
minimap, v4.16.0
pdf-view, v0.34.0
pigments, v0.18.1
Repl, v0.5.0
terminal-plus, v0.14.1

# Dev
No dev packages
```

**Comment by jeremyramin** [comment]
I'll take a look. Thanks for reporting this!

**Comment by jeremyramin** [comment]
Two terminal views should not be able to be open at the same time which makes this rather interesting. Either way, I have dealt with the situation where this error would occur.

Fixed as of v0.14.2
